incuna-sass/_normalize.sass

```diff
@@ -44,7 +44,7 @@ html
 // Links
 
 // Remove the gray background color from active links in IE 10.
-@if support-legacy-browser("ie", "10")
+.lt-ie11 &
```

Comment:
ok, so what's the solution to that then? I can remove the class, if it's just removing a bg color it shouldn't break anything anyway. Or I can take the whole thing out?